src/map/ai/controllers/automaton_controller.cpp

```diff
@@ -223,9 +223,9 @@ void CAutomatonController::DoCombatTick(time_point tick)
 
 void CAutomatonController::Move()
 {
-    float currentDistance = distanceSquared(PAutomaton->loc.p, PTarget->loc.p);
+    const float currentDistance = distance(PAutomaton->loc.p, PTarget->loc.p);
 
-    if ((shouldStandBack() && (currentDistance > 225)) || (PAutomaton->health.mp < 8 && PAutomaton->health.maxmp > 8))
+    if ((shouldStandBack() && (currentDistance > 15)) || (PAutomaton->health.mp < 8 && PAutomaton->health.maxmp > 8))
```

Comment:
Should this be checking `!isWithinDistance(PAutomaton->loc.p, PTarget->loc.p, 15.0f)` instead of using `distance`?